### ISSUE TITLE
Create a Liquid module

### ISSUE THREAD

**Issue by daveaglick** [statiqdev/Statiq#10]
Takes Liquid template content and transforms it to output content. Use DotLiquid. Make the metadata, completed documents, and execution context available similar to how the Razor module does.

**Comment by daveaglick** [comment]
Still leaning towards DotLiquid given longer history, but there's now also Fluid: https://github.com/sebastienros/fluid

**Comment by jbrinkman** [comment]
Is this still desired?  I have some experience with DotLiquid and some time available.

**Comment by daveaglick** [comment]
@jbrinkman Yes, very much so. I've long been a fan of Liquid (and DotLiquid) but there always seems to be something more pressing on my plate. If you're interested in taking a look, please do! I'd take a look at the Markdown extension for an example of a similarly scoped templating module and how it might work.

**Comment by laedit** [comment]
Is this not done through https://github.com/statiqdev/Statiq.Framework/pull/120 since scriban is compatible with liquid?
